Tests/testrunner.cpp

```diff
@@ -39,62 +39,70 @@
 /// Implementation of the googletest testrunner
 int main(int argc, char* argv[])
 {
+    std::string logLevel("all");
```

Comment:
Maybe it would be better to read this setting from the environment, cf. `std::getenv`. This way we would not interfere with gtest's parameters.